scripts/core/context.yml

```diff
@@ -81,8 +81,10 @@ params:
       name: phDevices
       desc: |
             [in][optional][range(0, numDevices)] array of device handles which context has visibility.
-            if nullptr, then all devices supported by the driver instance are visible to the context.
-            otherwise, context only has visibility to devices in this array.
+            if nullptr, then all devices and any sub-devices supported by the driver instance are
+            visible to the context.
+            otherwise, the context only has visibility to the devices and any sub-devices of the
```

Comment:
nit: Upper case in O in `otherwise`.

Reply:
The capitalization is the same as it was before this PR.  If we're going to change capitalization I'd prefer to change it uniformly and everywhere.

Reply:
we can do it in other PR.